Description: add mojo interface file that is missing from the tarball after https://chromium.googlesource.com/chromium/src/+/2b09b8221d07f6a13569f0358c6dbc721bed2d59.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- /dev/null
+++ b/chrome/test/data/webui/mojo/foobar.mojom
@@ -0,0 +1,16 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Test Mojo interfaces for MojoWebUIControllerBrowserTest.
+
+module test.mojom;
+
+interface Foo {
+  GetFoo() => (string value);
+};
+
+interface Bar {
+  GetBar() => (string value);
+};
+
